app/views/sessions/new.html.erb

```diff
@@ -1,15 +1,26 @@
-<h1>Login</h1>
-
-<%= form_tag sessions_path do |form| %>
-  <div class="field">
-    <%= label_tag :email %>
-    <%= text_field_tag :email %>
-  </div>
-  <div class="field">
-    <%= label_tag :password %>
-    <%= password_field_tag :password %>
+<section class="terms-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto">
```

Comment:
When it is success, would be it better if it disappear after X seconds ?
Also, would it better if it's more green? 

![image](https://user-images.githubusercontent.com/8007722/71861449-6e459a80-30c4-11ea-9a21-45cd0df77a5d.png)

Reply:
It should be green, I'll check it out.